multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpcomics/WPComics.kt

```diff
@@ -288,6 +291,13 @@ abstract class WPComics(
         "xuyen-khong" to "Xuyên Không",
     )
 
+    override fun getFilterList(): FilterList {
```

Comment:
These would now also show up in XoxoComics where they do nothing. You could add an override in XoxoComics with appropriate filters or at least an emptylist so that these don't show up.

Reply:
okie

Reply:
XoxoComics is not actually using WPComics generator. Should I create another PR to move it out of multisrc?

Reply:
What do you mean it isn't?

Reply:
Sorry, just my misunderstanding. I'm implementing XoxoComics's filter.